include/swift/AST/DiagnosticsSema.def

```diff
@@ -4657,13 +4657,24 @@ ERROR(availability_decl_unavailable, none,
       "%select{ in %3|}2%select{|: %4}4",
       (unsigned, DeclName, bool, StringRef, StringRef))
 
+WARNING(availability_decl_unavailable_warn, none,
+        "%select{getter for |setter for |}0%1 is unavailable"
```

Comment:
Please add both of these new diagnostics to `localization/diagnostics/en.yaml`

Reply:
Oh neat! Will do.